collective/trustedimports/suds.rst

```diff
@@ -0,0 +1,16 @@
+Zip
+===
+
+When used in RestrictedPython we can still use Suds
+
+>>> teval("from suds.client import Client; from suds import WebFault,MethodNotFound;return Client('')")
```

Comment:
How are you testing the result of a client call and subaccess to it?
e..g https://webkul.com/blog/python-suds-client/